app.py

```diff
@@ -1,41 +1,54 @@
+from _pytest.assertion import pytest_assertrepr_compare
 from flask import Flask, render_template, request, redirect, url_for
 import ApiAccess as api
+import View_Model as model
 
 
-app = Flask(__name__)
-app.config.from_object('flask_config.Config')
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object('flask_config.Config')
+    global obj2
+    obj2 = model.ViewModel()
+    return app
 
-obj1 = api.AccessTrelloApi()
 
-
-def get_items():
-
-    Items1 = obj1.getCardsFromTrelloList(
-        api.TODOLISTURL, 'To Do')
-
-    Items2 = obj1.getCardsFromTrelloList(
-        api.DONELISTURL, 'Done')
-
-    return Items1 + Items2
+app = create_app()
 
 
 @app.route('/')
 def index():
-    return render_template('index.html', items=get_items())
+    obj2.add_items()
```

Comment:
Rather than using this global object, we should instantiate a new `ViewModel()` in each route, which could then be populated when requested. This would prevent us needing to manage those as carefully between different endpoints